### DISCUSSION TITLE
xml-issue

### DISCUSSION THREAD

**Discussion by bethatasitmay** [PaloAltoNetworks/pan-os-php#661] [Q&A] [answered]
Hi

Is the xml-issue command as outlined in the wiki supposed to leave a local xml file?  When I run it I don't get a file (could be an issue with PowerShell).

**Comment by swaschkut** [comment, marked as answer]
thanks for finding this and reaching out:
no issue with powershell, actual pan-os API mode compare to out= argument is not working correct for type=xml-issue


work around until it is fixed:
`pan-os-php type=upload in=api://MGMT-IP out=configfile.xml`
`pan-os-php type=xml-issue in=configfile.xml out=xmlissue_final.xml`

already fixed in Docker develop container; will be published with version 2.0.59

**Comment by bethatasitmay** [comment]
Thanks!
